ingestion/src/metadata/ingestion/ometa/client.py

```diff
@@ -184,17 +184,13 @@ def _request(
             "verify": self._verify,
         }
 
-        masked_opts = self._mask_authorization_headers(opts)
-
         method_key = "params" if method.upper() == "GET" else "data"
         opts[method_key] = data
 
         total_retries = self._retry if self._retry > 0 else 0
         retry = total_retries
         while retry >= 0:
             try:
-                logger.debug("URL %s, method %s", url, method)
-                logger.debug("Data %s", masked_opts)
```

Comment:
this runs on EVERY request from the client. We don't need that much information on debug.